src/synthesizer/parametric/sf_hist.py

```diff
@@ -536,3 +571,94 @@ def _sfr(self, age):
         term1 = (age / self.tau) ** self.alpha
         term2 = (age / self.tau) ** self.beta
         return (term1 + term2) ** -1
+
+
+class DenseBasis(Common):
+    """
+    Dense Basis representation of a SFZH.
```

Comment:
Is it a SFZH or SFH? Everything else would imply SFH.

Reply:
SFH, typo, fixed